packages/dx-react-chart/src/types/templates.series.types.ts

```diff
@@ -0,0 +1,110 @@
+import {
+  Scales, BuildAnimatedStyleGetterFn, TransformedPoint,
+} from '@devexpress/dx-chart-core';
+import { PureComputed } from '@devexpress/dx-core';
+
+type Path = PureComputed<[TransformedPoint[]], string>;
+
+interface InternalPathProps {
+  /** @internal */
+  index: number;
+  /** @internal */
+  state?: string;
+  /** @internal */
+  style?: any;
+  /** @internal */
+  scales: Scales;
+  /** @internal */
+  getAnimatedStyle: BuildAnimatedStyleGetterFn;
+  /** @internal */
+}
+
+interface InternalPointProps {
+/** @internal */
+  argument: any;
```

Comment:
Why is `value` field absent?

Reply:
It is internal properties. This interface extends RawPointProps, RawSliceProps and RawBarProps interfaces and they have `value` prop.